src/backend/access/transam/xloginsert.c

```diff
@@ -149,9 +149,6 @@ static XLogRecData *XLogRecordAssemble(RmgrId rmid, uint8 info,
 static bool XLogCompressBackupBlock(char *page, uint16 hole_offset,
 									uint16 hole_length, char *dest, uint16 *dlen);
 
-/* Timeout in milliseconds for delaying WAL inserts to avoid WAL overflow */
-#define MB ((XLogRecPtr)1024*1024)
```

Comment:
Commit 87cdba56dc69552b08b8f231f788c3306e7046af moved away the code that the comment referred to:
```
@@ -122,7 +122,6 @@ static bool XLogCompressBackupBlock(char *page, uint16 hole_offset,
                                                                        uint16 hole_length, char *dest, uint16 *dlen);
 
 /* Timeout in milliseconds for delaying WAL inserts to avoid WAL overflow */
-#define BACK_PRESSURE_TIMEOUT 100
 #define MB ((XLogRecPtr)1024*1024)
 
 /*
```